src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -96,11 +101,11 @@
         :label               (i18n/label :t/share-collectible)
         :right-icon          :i/external}]]]))
 
-(defn f-view-internal
-  [{:keys [theme] :as _props}]
+(defn view-internal
+  [_]
   (let [selected-tab  (reagent/atom :overview)
         on-tab-change #(reset! selected-tab %)]
-    (fn []
+    (fn [{:keys [theme] :as _props}]
```

Comment:
An additional fix, we were receiving the `theme` in the wrong place, the component wouldn't re-render if it changed

Reply:
can also use the hook directly, if you prefer 👍